src/main/java/org/wso2/carbon/module/swiftiso20022/models/mt940models/RequestPayloadModel.java

```diff
@@ -57,6 +57,9 @@ public String getBlock2() {
     public void setBlock3(String block3) {
         this.block3 = block3;
     }
+    public String getBlock3() {
```

Comment:
```suggestion
    }
    
    public String getBlock3() {
```

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/8076e9b5033a9608b6839de38320ec994e01055d